light_painting/launch/t1_initialize.launch

```diff
@@ -4,12 +4,12 @@
 
     <!-- Launch Parameters -->
     <arg name="sim" default="true" />
-    <arg name="arduino_port" default="/dev/ttyACM0" />
+    <!-- <arg name="arduino_port" default="/dev/ttyACM0" /> -->
 
 
     <!-- Arduino Connection (only if on real hardware) -->
-    <node pkg="rosserial_python" type="serial_node.py" name="serial_node" respawn="true" unless="$(arg sim)" >
-        <param name="port" value="$(arg arduino_port)" />
+    <node pkg="light_painting" type="serial_node.py" name="serial_node" respawn="false"  >
```

Comment:
I'd suggest respawning this (ie. set `True`) so that the node will reconnect if it catches a bug and breaks the connection.